### PR TITLE
Migrate from "webexteamssdk" library to "webexpythonsdk" library

### PR DESCRIPTION
**Pull request by Sakthivel-Ramasamy**
Migration from "webexteamssdk" library to "webexpythonsdk" library.

All instances of package name "webexteamssdk" is replaced with "webexpythonsdk". All instances of primary API object "WebexTeamsAPI" is replaced with "WebexAPI". All instances of access token environment variable "WEBEX_TEAMS_ACCESS_TOKEN" is replaced with "WEBEX_ACCESS_TOKEN".

Changes are inline with "https://webexcommunity.github.io/WebexPythonSDK/user/migrate.html" and tested with my local Webex bot (which resulted in no errors). Only thing is all instances of "webexteamssdk" references in local Webex bot had to be replaced with "webexpythonsdk".

### CODE REVIEW

File: requirements_dev.txt [hunk 1]
```diff
@@ -7,6 +7,6 @@ tox==3.14.0
 coverage==4.5.4
 Sphinx==1.8.5
 twine==1.14.0
-webexteamssdk==1.6.1
+webexpythonsdk
```
Comment (fbradyirl) [inline requirements_dev.txt:10]: Please pin to latest version

File: setup.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 with open('README.md') as readme_file:
     readme = readme_file.read()
 
-requirements = ['webexteamssdk==1.6.1', 'coloredlogs', 'websockets==11.0.3', 'backoff']
+requirements = ['webexpythonsdk', 'coloredlogs', 'websockets==11.0.3', 'backoff']
```
Comment (fbradyirl) [inline setup.py:10]: Please pin to latest version

**Review by fbradyirl — COMMENTED** [review]
Thanks for the PR. 

Could you run the bot locally and post some logs here to verify sending and receiving messages is ok with this code change. Both plain text and adaptive cards. Thanks.

**Comment by fbradyirl** [comment]
Python version is also now updated to Python 3.12 in main branch.

File: requirements_dev.txt (hunk 1 above)
Comment (Sakthivel-Ramasamy) [reply requirements_dev.txt]: Pointed webexpythonsdk to the latest version (2.0.2)

File: setup.py (hunk 2 above)
Comment (Sakthivel-Ramasamy) [reply setup.py]: Pointed webexpythonsdk to the latest version (2.0.2)

**Comment by Sakthivel-Ramasamy** [comment]
Thanks for updating the Python version in main branch

**Comment by Sakthivel-Ramasamy** [comment]
[webex_bot_log_file.log](https://github.com/user-attachments/files/18206147/webex_bot_log_file.log)

Log file attached. No issues noticed with the library change and code changes!

**Pull request merged**